applications/nrf5340_audio/src/bluetooth/ble_core.c

```diff
@@ -267,3 +273,162 @@ int ble_core_init(ble_core_ready_t ready_callback)
 
 	return 0;
 }
+
+static void shell_help_print(const struct shell *shell)
+{
+	shell_print(shell, "Possible TX power values:");
+	shell_print(shell, "\tzero: 0dBm");
+	shell_print(shell, "\tneg1: -1dBm");
+	shell_print(shell, "\tneg2: -2dBm");
+	shell_print(shell, "\tneg3: -3dBm");
+	shell_print(shell, "\tneg4: -4dBm");
+	shell_print(shell, "\tneg5: -5dBm");
+	shell_print(shell, "\tneg6: -6dBm");
+	shell_print(shell, "\tneg7: -7dBm");
+	shell_print(shell, "\tneg8: -8dBm");
+	shell_print(shell, "\tneg12: -12dBm");
+	shell_print(shell, "\tneg16: -16dBm");
+	shell_print(shell, "\tneg20: -20dBm");
+	shell_print(shell, "\tneg40: -40dBm");
+}
+
+static enum ble_hci_vs_tx_power tx_power_input_check_and_enum_return(const char *input)
+{
+	if (strcmp(input, "zero") == 0) {
```

Comment:
Can this be made simpler by using e.g. "-2" instead of "neg2"?

Reply:
I assumed this was hard to do, because it was not done in the previous implementation we had of the TX power shell commands.
But I can do more investigation if desired.

Reply:
Changed to "-2" etc. while still using strcmp